UPGRADING.md

```diff
@@ -1,6 +1,42 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.3.4
+
+#### Reworking stream and file and un-deprecating stream like-objects
+
+Previously in 0.16 stream-like objects were deprecated. This release restores their functionality for use-cases other than file streaming.
+
+For streaming files, simply use file always.
+
+```ruby
+class API < Grape::API
+  get '/' do
+    file '/path/to/file'
+  end
+end
+```
+
+If you want to stream other kinds of content from a streamer object you may.
```

Comment:
Or shorter ...

Use `stream` to stream other kinds of content. In the following example a streamer class ...